.pre-commit-config.yaml

```diff
@@ -26,31 +26,36 @@ repos:
       exclude: tests/
       args: ["--profile", "black"]
 
-- repo: https://github.com/asottile/pyupgrade
-  rev: v3.19.1
-  hooks:
-    - id: pyupgrade
-
 - repo: https://github.com/psf/black
   rev: 24.10.0
   hooks:
     - id: black
       exclude: tests/
       args: [--line-length=100]
 
+- repo: https://github.com/pre-commit/mirrors-autopep8
+  rev: v1.5.7
+  hooks:
+    - id: autopep8
+      exclude: tests/
+      args: [--max-line-length=100, --aggressive, --aggressive]
+
 - repo: https://github.com/PyCQA/flake8
   rev: 7.1.1
   hooks:
     - id: flake8
       exclude: tests/
-      args: [--max-line-length=100 ]
+      args: [--max-line-length=100]
+
+- repo: https://github.com/asottile/pyupgrade
+  rev: v3.19.1
+  hooks:
+    - id: pyupgrade
 
 - repo: https://github.com/PyCQA/bandit
   rev: '1.8.0'
   hooks:
     - id: bandit
-      exclude: tests/
-      args: ["--skip", "B105"]
```

Comment:
I did skip those, because we do have hardcoded client id and secret in our code, and the pre-commit hook will fail in this case

Reply:
so where did you put the id and the secret now?

Reply:
We can "ignore" this by putting a `#nosec` tag on the line. This works great for example code. Also, for our library we have a public secret key used by the Tado web app. No very hard secrets here, so we can apply that tag, I would say.